ios/AppDelegate.mm

```diff
@@ -72,6 +72,23 @@ -(void) handleMediaServicesWereReset:(NSNotification *)notification {
 }
 
 -(BOOL) application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+
+	int argc = 1;
+	char *argv[3] = { 0 };
+	NSURL* nsUrl = [launchOptions objectForKey:UIApplicationLaunchOptionsURLKey];
+
+	if (nsUrl != nullptr) {
+		NSString *nsString = nsUrl.absoluteString;
+		const char *string = nsString.UTF8String;
+		string += strlen("file://");
```

Comment:
Should we check if it does actually start with `file://`?  Could we end up with an actual http:// URL?

-[Unknown]

Reply:
>Should we check if it does actually start with file://? 

fixed

>Could we end up with an actual http:// URL?

http is not registered as a known url. It should be handled separately if needed